Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ghcr.io/linuxserver/baseimage-alpine:3.16
```

Comment:
Use 3.15 for now until 3.16 is fully switched over to s6 v3 otherwise it'll make your life difficult transitioning the container.

Reply:
Good call

Reply:
Resolved in https://github.com/linuxserver/docker-fail2ban/pull/1/commits/6878c98e634358f351f8f1f41e54e96ca859fdb0